test/sandbox/test/layouts_test.rb

```diff
@@ -31,7 +31,7 @@ class LayoutsTest < ActionDispatch::IntegrationTest
     test "rendering with_content_for" do
       get "/layout_with_content_for"
       assert_response :success
-      assert_select 'body[data-layout="with_content_for"]', "Hello content for\n\n  Foo: bar"
+      assert_select 'body[data-layout="with_content_for"]', "Hello content for\n  Foo: bar"
```

Comment:
I find this change a bit confusing. What about adding a dedicated test to be more explicit about what's the expected behavior?